cmd/fleetctl/preview.go

```diff
@@ -30,7 +30,7 @@ import (
 
 const (
 	downloadUrl             = "https://github.com/fleetdm/osquery-in-a-box/archive/%s.zip"
-	standardQueryLibraryUrl = "https://raw.githubusercontent.com/fleetdm/fleet/main/docs/01-Using-Fleet/standard-query-library/standard-query-library.yml"
+	standardQueryLibraryUrl = "https://raw.githubusercontent.com/fleetdm/fleet/main/docs/Using-Fleet/standard-query-library/standard-query-library.yml"
```

Comment:
What about people on an older version of `fleetctl`?  What happens when we make this change to GitHub?  Will those folks stop being able to use `fleetctl preview`?  As long as there is either:
1. a good error message that fleetctl shows asking the user to update fleetctl (AND if the new version of fleetctl is actually published as soon as we merge this)
2. or `fleetctl preview` just fails gracefully and doesn't include the standard queries (AND the new version of fleetctl is actually published)

Then I'm cool with this.

But I think we might need to think through a different solution.  e.g. We could leave the file in the old location for a while.  @zwass up to you